### ISSUE TITLE
how does redex play within gradle builds ?

### ISSUE THREAD

**Issue by stephanenicolas** [facebook/redex#50]
Is there any way to get a gradle plugin to make redex widely used ?

**Comment by bertmaher** [comment]
I would love for someone to create this, but we don't use gradle at FB, so we've got no expertise to do it at the moment.

**Comment by conzorz** [comment]
Yeah, this library sounds great but no idea how to get it working on windows...

**Comment by githubdoramon** [comment]
@conzorz Best way to make it work on windows... create a virtual machine with linux on it hehehehe

**Comment by stephanenicolas** [comment]
And on DOS ? :D

Le mer. 13 avr. 2016 17:14, githubdoramon <notifications@github.com> a
écrit :

**Comment by shmuelr** [comment]
Hmm it should be very doable with a gradle script..
post compile in release variant > run redex on apk file

**Comment by timmutton** [comment]
Check out https://github.com/timmutton/redex-plugin

**Comment by bertmaher** [comment]
@timmutton that's awesome!

**Comment by shaunkawano** [comment]
@bertmaher Just for curiosity, what do you guys use at FB instead of Gradle? 😮

**Comment by bertmaher** [comment]
We use buck:  https://buckbuild.com/  :-)
